src/pages/gift-certificate/certificate-checkbox/certificate-checkbox.js

```diff
@@ -0,0 +1,65 @@
+import { Checkbox } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import React, { useState } from 'react';
+import NumberInput from '../../../components/number-input';
+import { getFromLocalStorage } from '../../../services/local-storage.service';
+import { INITIAL_CERTIFICATE_COUNT, LIGHT_THEME, MATERIAL_UI_COLOR } from '../../../configs';
+import { useStyles } from './certificate-checkbox.styles';
+
+const CertificateCheckbox = ({
+  handleAllCheckboxesChange,
+  handleAllCountChange,
+  index,
+  value,
+  checked
+}) => {
+  const styles = useStyles();
+  const { t } = useTranslation();
+  const theme = getFromLocalStorage('theme');
+
+  const checkTheme = () => {
+    if (theme === LIGHT_THEME) {
+      return MATERIAL_UI_COLOR.PRIMARY;
+    }
+    return MATERIAL_UI_COLOR.DEFAULT;
+  };
```

Comment:
we check our theme in palette that you are passing to makeStyles hook. 
And the question is: do we need to use inline styles in Checkbox component on the line 42 or we can set out styles in certificate-checkbox.styles file and use palette there?